ui/src/App.tsx

```diff
@@ -55,19 +57,15 @@ function App() {
           <div className="carousel-container">
             <div className="photo-carousel" data-preload>
               <Carousel autoplay dots={false}>
-                {loading ? (
-                  <Spin />
-                ) : (
-                  photos.map((photo) => {
-                    return (
-                      <div key={photo.src} className="container-fluid">
-                        <div className="content">
-                          <img alt={photo.alt} src={photo.src} />
-                        </div>
+                {photos.map((photo) => {
+                  return (
+                    <div key={photo.src} className="container-fluid">
+                      <div className="content">
+                        <img alt={photo.alt} src={photo.src} />
                       </div>
-                    );
-                  })
-                )}
+                    </div>
+                  );
+                })}
```

Comment:
why did you remove the the loading?

Reply:
I was told to remove the Spin. Should I put it back?

Reply:
👍  I wasn't aware of that